_pages/docs/quick-start-guide/installing.md

```diff
@@ -31,7 +31,7 @@ Install [Docker Compose](https://docs.docker.com/compose/install/)
 
 `docker-compose up -d`
 
-The designer will be running on `http://localhost:4200`, and the server on `http://localhost:5000`.
+The designer will be running on [http://localhost:4200](http://localhost:4200), and the server on [http://localhost:5000](http://localhost:5000).
```

Comment:
Don't forget about here!

Reply:
Oh yes, just noticed I'll need to do it in multiple spots - will push updates!